app/controllers/ApplicationController.scala

```diff
@@ -54,6 +54,20 @@ class ApplicationController @Inject()(val config: ApplicationConfig,
         Redirect(controllers.routes.ApplicationController.signedOut).withNewSession
     }
   }
+  def timedOut(service: String): Action[AnyContent] = Action {
```

Comment:
There is lot of code duplication here. I would just have one case for AWRS and the default case should call signout method